build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+SOURCE_DIR=$(readlink -f "${BASH_SOURCE[0]}")
+cd "$(dirname "$SOURCE_DIR")" || exit
+set -xeuo pipefail
+
+# Error below : can't install graphql proper version
```

Comment:
What's up w/ this?

Reply:
was trying to get typewriter or something and it threw odd things